app/models/admin_user.rb

```diff
@@ -0,0 +1,7 @@
+class AdminUser < ActiveRecord::Base
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable and :omniauthable
+  devise :database_authenticatable, :trackable, :validatable
+
+  validates :email, presence: true
```

Comment:
We can configure Active Admin to be accessed by a `User` model, what is the reason to use this new model `AdminUser`?